substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/ClassEntry.java

```diff
@@ -369,4 +370,48 @@ public MethodEntry ensureMethodEntryForDebugRangeInfo(DebugRangeInfo debugRangeI
     public List<MethodEntry> getMethods() {
         return methods;
     }
+
+    public int lowpc() {
+        assert isPrimary();
+        return primaryEntries.get(0).getPrimary().getLo();
+    }
+
+    public int lowpcDeopt() {
```

Comment:
If you instead make `boolean` field `com.oracle.objectfile.debugentry.ClassEntry#includesDeoptTarget` into an `int` that is `-1` if there are no deoptTarget entries, of the index of the first deoptTarget entry if there are any then you can also avoid the loop here and just access the first deoptTarget entry to get its `lo` value. Would that work?

Reply:
That is a nice idea but it will be rendered unnecessary when I introduce a separate list for deopt methods.